### PR TITLE
limit word-input on ASC

### PR DESCRIPTION
**Pull request by tomlovesgithub**

### CODE REVIEW

**Review by warrensearle — COMMENTED** [review]
This looks good 👍 
Is this re-using the word limit implemented in QTs Scenario test? Or can Scenarios be updated so we share the same solution

**Comment by tomlovesgithub** [comment]
This is a new solution, i wasn't sure if the QT solution would fit - defo worth comparing them

**Review by warrensearle — APPROVED** [review]
Approved. As discussed please consider combining with the word count in QT Scenarios so we avoid having duplicate code and it's easier to maintain

**Review by mbrookeswebdev — APPROVED** [review]

**Comment by HalcyonJAC** [comment]
Hi @tomlovesgithub ,

First review here, hope I'm doing it right.
I've pulled down the branch and had a look at your work. It seems that it doesn't like it when you interact with the radio boxes. Once I select an answer it complains. Below there's a screenshot from the console.

Screenshot 2021-03-02 at 14 39 16

File: src/components/Form/FormField.vue [hunk 1]
```diff
@@ -129,6 +128,19 @@ export default {
             this.setError(this.pattern.message);
           }
         }
+
+        if (this.wordLimit) {
+          const words = this.value
```
Comment (HalcyonJAC) [inline src/components/Form/FormField.vue:133]: `this.value` can be `null` and the code evaluates to `null.replace()..` which makes the console yell :(

**Review by HalcyonJAC — CHANGES_REQUESTED** [review]

**Review by HalcyonJAC — APPROVED** [review]

**Pull request merged**